app/models/form/sales/questions/household_wheelchair_check.rb

```diff
@@ -9,15 +9,7 @@ def initialize(id, hsh, page)
       "0" => { "value" => "Yes" },
       "1" => { "value" => "No" },
     }
-    @hidden_in_check_answers = {
-      "depends_on" => [
-        {
-          "wheel_value_check" => 0,
-        },
-        {
-          "wheel_value_check" => 1,
-        },
-      ],
-    }
+    @hidden_in_check_answers = true
```

Comment:
Wound't we want to show it in check answers if it's not answered?

Reply:
oh - I would have assumed what @kosiakkatrina did too - might have to change some other interruption screens I've made if this is the case